src/client/components/Dropdown/DropDown.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Skills, StudyGroup, DropDown } from './DropDown';
+
+export default { title: 'Input' };
+
+export const StudyGroupDropdown = () => <StudyGroup title="Elastic" />;
```

Comment:
This title is actually the `defaultValue` or `placeholder`, not `title`. Minor nitpick.